TestCases/rans/flatplate/turb_KE_flatplate.cfg

```diff
@@ -0,0 +1,388 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Sort of.  The two files use different options.  `turb_v2f_flatplate.cfg` is just a cp of `turb_SA_flatplate.cfg` aside from the model used, while `turb_KE_flatplate.cfg` was used to test different approaches (e.g., `TIME_STEPPING`) during v2-f development.  I've deleted `turb_KE_flatplate.cfg` since it isn't used for anything at the moment.